bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/InitializeToolsStartup.java

```diff
@@ -99,27 +99,19 @@ public void run()
 			Logger.log(MessageFormat.format("esp-idf.json file doesn't exist at this location: '{0}'", url.getPath()));
 			return;
 		}
-		else if (isInstallerConfigSet())
+		else if (isConfigFileUpdated(idf_json_file))
```

Comment:
Hi @alirana01, what will happen, if idf_json_file is modified and we click `NO`? It looks like if we store preferences like that: 
`prefs.putLong(LAST_MODIFIED_ESP_IDF_JSON_FILE, idf_json_file.lastModified());` then this condition returns false after second eclipse restart and we will update our environment with new idf_json_file anyway `updateEnvUsingIdfJsonFile(idf_json_file);`. It's not clear to me, to be honest, can you explain?